nav2-usability/scenario_code/lib/deeplink-pathparam/deeplink_pathparam_go_router_prototype.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2022, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:go_router_prototype/go_router_prototype.dart';
+
+void main() {
+  runApp(BooksApp());
+}
+
+class Book {
+  final String title;
+  final String author;
+
+  Book(this.title, this.author);
+}
+
+class BooksApp extends StatelessWidget {
+  final List<Book> books = [
+    Book('Stranger in a Strange Land', 'Robert A. Heinlein'),
+    Book('Foundation', 'Isaac Asimov'),
+    Book('Fahrenheit 451', 'Ray Bradbury'),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp.router(
+      routeInformationParser: _router.parser,
+      routerDelegate: _router.delegate,
```

Comment:
Should these be passing the new route information provider?

```suggestion
      routeInformationProvider: _router.routeInformationProvider,
      routeInformationParser: _router.parser,
      routerDelegate: _router.delegate,
```

Reply:
This isn't supported in the prototype package right now